app/views/students/show.html

```diff
@@ -0,0 +1,125 @@
+<div ng-controller='StudentsShowCtrl as controller'>
+	<!-- Page Heading -->
+	<div class="row">
+		<div class="col-xs-12">
+			<h2 class="page-header">
+        <span translate="i18nStudentsPanelSection"></span>
```

Comment:
Try to create another translation with the same text. I know, maybe it sounds kind of weird, but let me explain you.

Your string representing your translation should represent where you will use it, no matter what its text is. So, in this case, I would define a translation
` i18nStudents: Alumnos`. 

Trade off of this option: cleaner and semantic code, and independent key-value translation per view vs. wording repetition.